bot/utils/checks.py

```diff
@@ -1,8 +1,8 @@
+from bot.config import devs
+
 from discord import Member
 from discord.ext.commands import Context, NoPrivateMessage
 
```

Comment:
```suggestion
from discord import Member
from discord.ext.commands import Context, NoPrivateMessage

from bot.config import devs

```
Wrong import order : local import should be located after third party import